core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7535,6 +7558,8 @@ public static final class Name {
     public static final String MASTER_METADATA_SYNC_UFS_PREFETCH_TIMEOUT =
         "alluxio.master.metadata.sync.ufs.prefetch.timeout";
     public static final String MASTER_METASTORE = "alluxio.master.metastore";
+    public static final String MASTER_INODE_METASTORE = "alluxio.master.inode.metastore";
+    public static final String MASTER_BLOCK_METASTORE = "alluxio.master.block.metastore";
```

Comment:
@elega Do you prefer `master.inode.metastore` or `master.metastore.inode`?

Reply:
i prefer metastore.inode

Reply:
@bzheng888 can you pls change the property keys accordingly? Thanks!

Reply:
Done